ingestion_server/ingestion_server/indexer.py

```diff
@@ -377,6 +387,9 @@ def go_live(write_index, live_alias):
         else:
             es.indices.put_alias(index=write_index, name=live_alias)
             log.info(f"Created '{live_alias}' index alias pointing to {write_index}")
+        # If there were any active workers for this task, we should mark them complete.
+        if active_workers:
+            active_workers.value = int(False)
```

Comment:
It looks like this is the only place where the `active_workers` value is "turned off", is that correct?

Reply:
Yes, at the moment. I was trying to account for the fact that the data refresh isn't really "done" until the `go_live` step completes, which is _after_ the workers finish and thus isn't accounted for if we turn this off as soon as the indexer workers complete.

We could introduce yet another global progress indicator to track just the status of the `go_live` step, which is turned on right before turning `active_workers` off, and turned off when the step completes. Alternatively we could continue considering this its own 'worker', so:
* turn `active_workers` _off_ when the last worker completes, _if_ any of them are in the Error state (because in this case we will not attempt to go live)
* turn `active_workers` off at the end of the `go_live` step as well

Reply:
Update -- done. The result is that:
* if workers fail, we do not attempt to `go_live`. When you query the refresh status it will correctly report that it is no longer active, and that there are errors
* if workers succeed, we do not consider the task inactive until `go_live` is complete

Reply:
That sounds perfect!